src/datagen/test_datagen.js

```diff
@@ -16,21 +16,17 @@
 
 /** @module datagen/test_datagen */
 /**
- * @fileoverview Contain functions that help in generating testSuites and
- * storing them in a file.
+ * @fileoverview Contains functions that help in generating testCases for
+ * validation of request header, request body and security requirements of an
+ * api endpoint and creating a testSuite file.
  */
 
+const {snakeCase} = require('snake-case');
```

Comment:
What's the use of this library?

Reply:
Created a custom function which does the same work.
[https://github.com/googleinterns/OAS-Adherence-Test-Suite/pull/4](url)